### PR TITLE
NMS-13428: Strings with URL arguments are truncated in the eventdescr field

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ x] Have you made a comment in that issue which points back to this PR?
* [ x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ x] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13428

### CODE REVIEW

File: docs/modules/operation/pages/events/sources/send-event.adoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
+
+[[send-event]]
+= send-event.pl utility
+
+The send-event.pl utility lets you create and send an event to {page-component-title} without using the OpenNMS component.
```
Comment (fooker) [inline docs/modules/operation/pages/events/sources/send-event.adoc:5]: > [...] without using the OpenNMS component.

What do you mean by this? Maybe something like 
... to trigger processes inside of OpenNMS.

File: docs/modules/operation/pages/events/sources/send-event.adoc [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+It is most frequently used to reload daemon configuration without having to restart {page-component-title}.
+You can also use the send-event.pl utility as an automation tool to trigger events as part of continuous integration scripts.
```
Comment (fooker) [inline docs/modules/operation/pages/events/sources/send-event.adoc:7]:
```suggestion
You can also use the send-event.pl utility as an automation tool to trigger events as part of continuous integration scripts or other automated processes.
```

File: docs/modules/operation/pages/events/sources/send-event.adoc [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+| Print raw XML that is generated.
+
+| -V (--version)
+| Print version and exit successfully.
+
+|===
+
+Note that parameter descriptions use the percentage sign (%) as a replacement variable.
+If a literal % sign appears in your description, you must escape it with a %.
+For example:
```
Comment (fooker) [inline docs/modules/operation/pages/events/sources/send-event.adoc:89]: Add a link to event-configuration.adoc (section "Replacement Tokens") here?

File: docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc [hunk 4]
```diff
@@ -33,3 +33,7 @@
 | no
 | yes
 |===
+
+== send-event.pl utility
+
+Use the send-event.pl utility to
```
Comment (fooker) [inline docs/modules/operation/pages/operation/daemon-config-files/eventd.adoc:39]: This seems to to be a left-over.

**Review by fooker — CHANGES_REQUESTED** [review]

File: docs/modules/operation/pages/events/event-configuration.adoc [hunk 5]
```diff
@@ -75,10 +75,16 @@ You can include various tokens in the description, log message, operator instruc
 These tokens will be replaced by values from the current event when the text for the event is constructed.
 Not all events will have values for all tokens, and some refer specifically to information available only in events derived from SNMP traps.
 
+Note that parameter descriptions use the percentage sign (%) as a replacement variable.
```
Comment (christianpape) [inline docs/modules/operation/pages/events/event-configuration.adoc:78]: The percentage sign itself is not a replacement variable. It is used as a kind of delimiter to mark a replaceable variable.

File: docs/modules/operation/pages/events/sources/send-event.adoc [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+| -v (--verbose)
+| Print raw XML that is generated.
+
+| -V (--version)
+| Print version and exit successfully.
+
+|===
+
+Note that parameter descriptions use the percentage sign (%) as a replacement variable.
```
Comment (christianpape) [inline docs/modules/operation/pages/events/sources/send-event.adoc:87]: Same here. The percentage sign itself is not a replacement variable. It is used as a kind of delimiter to mark a replaceable variable.

**Review by christianpape — CHANGES_REQUESTED** [review]

**Review by fooker — APPROVED** [review]

**Review by christianpape — APPROVED** [review]
Nice!

**Comment by Bonrob2** [comment]
Thanks for the help and the reviews!

**Pull request merged**